cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -56,6 +56,7 @@ import org.ossreviewtoolkit.utils.ORT_PACKAGE_CURATIONS_DIRNAME
 import org.ossreviewtoolkit.utils.ORT_PACKAGE_CURATIONS_FILENAME
 import org.ossreviewtoolkit.utils.ORT_REPO_CONFIG_FILENAME
 import org.ossreviewtoolkit.utils.expandTilde
+import org.ossreviewtoolkit.utils.log
```

Comment:
The commit message still says "applyRepositoryPackageCurations".

BTW, my general recommendation to not mention newly introduced properties by name in the commit message (for exactly the reason happened here), but rather say something more general like "Introduce a configuration flag for repository package curations".